src/Connectors/src/Connector/MongoDb/MongoDbTypeLocator.cs

```diff
@@ -5,7 +5,7 @@
 using System.Reflection;
 using Steeltoe.Common.Reflection;
 
-namespace Steeltoe.Connector.MongoDb;
+namespace Steeltoe.Connector.CosmosDb;
```

Comment:
I was going to add a suggestion to change this back to `MongoDb`, but it would impact other files too,  and it looks like all of the TypeLocators got moved into `Steeltoe.Connector.CosmosDb` regardless of what they're for